website/release-notes/sprint-review-2504.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Wazo Platform 25.04 Released
+date: 2025-03-17T08:00:00
+authors: wazoplatform
+category: Wazo Platform
+tags: [wazo-platform, development]
+slug: release-review-2504
+status: published
+---
+
+Hello Wazo Platform community!
+
+Here is a short review of the Wazo Platform 25.04 release.
+
+## New Features in This Release
+
+- **Recordings**: Recording announcement sounds can now be configured
+
+## Bug Fixes
+
+- **Ring Groups**: Ring group options such as `dtmf_record_toggle` or options inserted through
+  dialplan customization should now be properly applied to connected group members
+
+## Technical
+
+- **Auth**: A new python plugin entrypoint `wazo_auth.idp` has been added, enabling support for alternative
+  authentication mechanisms
```

Comment:
This is rude for a sprint review, without doc or example
IMO we could remove it

Reply:
The doc is WIP. 
We could omit it and make a separate blog post when the doc is ready.
Or mention it in the sprint review but mention that doc is coming.